Core/src/org/sleuthkit/autopsy/datamodel/GetSCOTask.java

```diff
@@ -58,33 +68,70 @@
     @Override
     public void run() {
         AbstractContentNode<?> contentNode = weakNodeRef.get();
-
         //Check for stale reference or if columns are disabled
         if (contentNode == null || UserPreferences.getHideSCOColumns()) {
             return;
         }
         // get the SCO  column values
-        List<Tag> tags = contentNode.getAllTagsFromDatabase();
-        Pair<Score, String> scoreAndDescription;  
-        DataResultViewerTable.HasCommentStatus comment;
+        Pair<Score, String> scoreAndDescription;
+        ;
         Pair<Long, String> countAndDescription = null;
+        scoreAndDescription = contentNode.getScorePropertyAndDescription();
 
-        scoreAndDescription = contentNode.getScorePropertyAndDescription(tags);
-        //getting the correlation attribute and setting the comment column is done before the eamdb isEnabled check
-        //because the Comment column will reflect the presence of comments in the CR when the CR is enabled, but reflect tag comments regardless
         String description = Bundle.GetSCOTask_occurrences_defaultDescription();
         List<CorrelationAttributeInstance> listOfPossibleAttributes = new ArrayList<>();
         Content contentFromNode = contentNode.getContent();
+        //the lists returned will be empty if the CR is not enabled
         if (contentFromNode instanceof AbstractFile) {
             listOfPossibleAttributes.addAll(CorrelationAttributeUtil.makeCorrAttrsForSearch((AbstractFile) contentFromNode));
         } else if (contentFromNode instanceof AnalysisResult) {
             listOfPossibleAttributes.addAll(CorrelationAttributeUtil.makeCorrAttrsForSearch((AnalysisResult) contentFromNode));
         } else if (contentFromNode instanceof DataArtifact) {
             listOfPossibleAttributes.addAll(CorrelationAttributeUtil.makeCorrAttrsForSearch((DataArtifact) contentFromNode));
-        } else {
-            //JIRA-TODO : add code for Jira-7938 OsAccounts
+        } else if (contentFromNode instanceof OsAccount) {
+            try {
+                List<OsAccountInstance> osAccountInstances = ((OsAccount) contentFromNode).getOsAccountInstances();
+
+                OsAccountInstance osAccountInstance = osAccountInstances.isEmpty() ? null : osAccountInstances.get(0);
+                /*
```

Comment:
Please move line 95 below the big comment and remove "which necessitates the following code block currently."